doc/releases/release-notes-2.0.rst

```diff
@@ -86,6 +86,16 @@ Architectures
     MSI and other features required for PCIe devices. The previous PCI
     implementation has been deprecated and will be removed in 2.1.
 
+* RISC-V:
+
+  * Added support for the SiFive HiFive1 Rev B development board.
+  * Added support for LiteX VexRISCv soft core.
+  * Added support for 64-bit RISC-V, renaming the architecture from "riscv32"
+    to "riscv".
+  * Added 64-bit Qemu support.
```

Comment:
``QEMU``

Reply:
Do you want me to fix the one in ARM's release notes too?

Reply:
Yes, please!

Reply:
Done!